homeassistant/components/reolink/util.py

```diff
@@ -51,6 +51,15 @@ def is_connected(hass: HomeAssistant, config_entry: config_entries.ConfigEntry)
     )
 
 
+def get_host(hass: HomeAssistant, config_entry_id: str) -> ReolinkHost:
+    """Return the Reolink host from the config entry id."""
+    config_entry: ReolinkConfigEntry | None = hass.config_entries.async_get_entry(
+        config_entry_id
+    )
+    assert config_entry is not None
```

Comment:
We should add proper error handling here as it could be that the config entry id is invalid

Reply:
Done, proper error handeling is added including a test for it.